### ISSUE TITLE
[Bug]: Strange condition on UserInfo

### ISSUE THREAD

**Issue by valadas** [dnnsoftware/Dnn.Platform#5977]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

![image](https://github.com/dnnsoftware/Dnn.Platform/assets/6371568/bab9301e-b2bf-49a8-aae6-58db4ec5c376)
This condition will always be false after refactoring in #5971

### Steps to reproduce?

This condition should never be entered as it will always be false, this is just a note to remove this block at some point in time.

### Affected Versions

9.13.3 (alpha)

### What browsers are you seeing the problem on?

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jeremy-farrance** [comment]
Is it just a typo here (".UserInfo" should be just ".User")?

![image](https://github.com/dnnsoftware/Dnn.Platform/assets/653301/79d95ac4-d979-46e5-9602-467c0fb4fd28)

**Comment by valadas** [comment]
No, basically this is the user profile module, it used to be used by both users and admins but now admins have a new experience in the persona bar. So this code was all refactored to no longer user `User` and only `UserInfo` and that block of code is basically to remove, it is right now always false and this is just a reminder for code cleanup at some point.

**Comment by Mostafa-Moafi** [comment]
> No, basically this is the user profile module, it used to be used by both users and admins but now admins have a new experience in the persona bar. So this code was all refactored to no longer user `User` and only `UserInfo` and that block of code is basically to remove, it is right now always false and this is just a reminder for code cleanup at some point.

Do you just have to remove the condition?

**Comment by valadas** [comment]
@Mostafa-Moafi basically the whole if block can go away as one will never be both a superuser and not a superuser, so that code will never ever run anyway.
